client/scripts/rails_only.jsx

```diff
@@ -8,3 +8,7 @@ require('es5-shim/es5-sham');
 // Due to issue: https://github.com/ariya/phantomjs/issues/12401
 // Phantomjs does not like promises
 require('es6-promise').polyfill();
+
+require('expose?jQuery!jquery');
+require('expose?$!jquery');
+require('expose?jquery-ujs!jquery-ujs');
```

Comment:
I don't understand this line. `jquery-ujs` is not a valid js variable name, and it not something to expose like this.

What *is* important is that the code get invoked, which means that it needs to be in an entry point.